simple_parsing/helpers/nested_partial.py

```diff
@@ -0,0 +1,48 @@
+import functools
+from typing import Any, Generic, TypeVar
+
+_T = TypeVar("_T")
+
+
+class npartial(functools.partial, Generic[_T]):
```

Comment:
Thanks for this! It would still be nice to have instance support in subgroups with frozen dataclasses.

Reply:
#223